extensions/amp-analytics/analytics-vars.md

```diff
@@ -206,6 +206,42 @@ is unloaded. The value is in milliseconds.
 
 Example value: `40`
 
+### navRedirectCount
+
+Provides the number of redirects since the last non-redirect navigation.
+See W3C Navigation Timing API [PerformanceNavigation interface](https://www.w3.org/TR/navigation-timing/#performancenavigation) for more information.
+
+Example value: `0`
+
+### navTiming
+
+Provides access to metrics from the browser's PerformanceTiming interface.
```

Comment:
Can you change start-event to startEvent and end-event to endEvent? that way, we'll have consistency in naming. Here and below.